AboutNewTabService.jsm

```diff
@@ -19,7 +19,7 @@ const TOPIC_CONTENT_DOCUMENT_INTERACTIVE = "content-document-interactive";
 
 // Automated tests ensure packaged locales are in this list. Copied output of:
 // https://github.com/mozilla/activity-stream/blob/master/bin/render-activity-stream-html.js
-const ACTIVITY_STREAM_BCP47 = "en-US ach an ar ast az be bg bn br bs ca cak crh cs cy da de dsb el en-CA en-GB eo es-AR es-CL es-ES es-MX et eu fa ff fi fr fy-NL ga-IE gd gl gn gu-IN he hi-IN hr hsb hu hy-AM ia id is it ja ja-JP-macos ka kab kk km kn ko lij lo lt ltg lv mk mr ms my nb-NO ne-NP nl nn-NO oc pa-IN pl pt-BR pt-PT rm ro ru si sk sl sq sr sv-SE ta te th tl tr trs uk ur uz vi zh-CN zh-TW".split(" ");
+const ACTIVITY_STREAM_BCP47 = "en-US".split(" ");
```

Comment:
I looked at the code but don't quite understand what this is for. Just confirming you intended to change it.

Reply:
Yeah, it's the line printed out when running `bundle:html`:
```
Writing prerendered files to individual directories under /Users/Ed/Development/activity-stream/prerendered:
 ✓ en-US 
 ✓ en-US (static) 
 Skipped the following locales because they use the same strings as en-US or its language locale: ach, af, an, ar, ast, az, be, bg, bn, br, bs, ca, cak, crh, cs, cy, da, de, dsb, el, en-CA, en-GB, eo, es-AR, es-CL, es-ES, es-MX, et, eu, fa, ff, fi, fr, fy-NL, ga-IE, gd, gl, gn, gu-IN, he, hi-IN, hr, hsb, hu, hy-AM, ia, id, is, it, ja, ja-JP-mac, ka, kab, kk, km, kn, ko, lij, lo, lt, ltg, lv, mk, mr, ms, my, nb-NO, ne-NP, nl, nn-NO, oc, pa-IN, pl, pt-BR, pt-PT, rm, ro, ru, si, sk, sl, son, sq, sr, sv-SE, ta, te, th, tl, tr, trs, uk, ur, uz, vi, wo, xh, zh-CN, zh-TW 

If aboutNewTabService tests are failing for unexpected locales, make sure its list is updated:
const ACTIVITY_STREAM_BCP47 = "en-US".split(" ");
```

It's saying only "en-US" is prerendered because all locales have files that would have prerendered the same way as "en-US" so use that one.